frontend/sw360-portlet/src/main/webapp/css/sw360.css

```diff
@@ -321,3 +321,8 @@ input[readonly].clickable {
 .release-list-entry-element input {
     width: 100%;
 }
+
+.hashvalue {
+    word-break: break-all;
+    font-family: monospace;
+}
```

Comment:
I guess that the class "checksumInput" is not used any more after removing these lines. Maybe we should remove it in the css, too.